htdocs/api/class/api_documents.class.php

```diff
@@ -472,6 +472,20 @@ public function getDocumentsListByElement($modulepart, $id = 0, $ref = '', $sort
 			}
 
 			$upload_dir = $conf->knowledgemanagement->dir_output.'/knowledgerecord/'.dol_sanitizeFileName($object->ref);
+		} elseif ($modulepart == 'ticket') {
+			require_once DOL_DOCUMENT_ROOT.'/ticket/class/ticket.class.php';
+
+			if (!DolibarrApiAccess::$user->rights->ticket->read && !DolibarrApiAccess::$user->rights->ticket->read) {
```

Comment:
@grandoc `!DolibarrApiAccess::$user->rights->ticket->read && !DolibarrApiAccess::$user->rights->ticket->read` there is twice the same right